repos/system_upgrade/common/libraries/repofileutils.py

```diff
@@ -11,6 +11,15 @@
     api.current_logger().warning('repofileutils.py: failed to import dnf')
 
 
+class InvalidRepoDefinition(Exception):
+    def __init__(self, msg, repofile=None, repoid=None):
```

Comment:
I am not sure if there is any benefit to have the default arguments here, "None" would look weird in the message. I would leave this up to the caller. Wdyt?

Reply:
You are right